.storybook/preview.tsx

```diff
@@ -41,6 +41,7 @@ const parameters: Parameters = {
 			return <DocsContainer {...props} />;
 		},
 	},
+	actions: { argTypesRegex: '^on[A-Z].*' },
```

Comment:
This will auto pickup and prop that starts with `on` as an action and display it in the actions addon